src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -41,15 +48,17 @@ export class ArticlesTable extends Component {
     super(props);
 
     this.state = {
-      deleteArticleId: null,
+      ...notesInitialState,
+      deleteArticleId: 0,
```

Comment:
`0` is a valid article id. If I understand correctly, I won't be able to delete it.

Reply:
:+1: 

`s/0/-1/`